### ISSUE TITLE
[BUG]: Color contrast for active links or blue text/font.

### ISSUE THREAD

**Issue by colabottles** [TechIsHiring/techishiring-website#155]
The current color for active links does not have sufficient color contrast that meets or exceeds the WCAG 1.4.3 Contrast (Minimum) guideline ratio of 4.5:1 (3.67:1)

## Expected Behavior

Contrast for active links (and all text) should be at a ratio of 4.5:1 or greater.

## Possible Solution

Use a deeper shade of blue for active links (or any blue text) that meets or exceeds the contrast guidelines. With users that have Trianopia (cannot see blues), they will see a aqua/teal color instead so that should meet or exceed contrast guidelines as well.

## Context (Environment)

Users with low-vision and Tritanopia currently will have difficulty seeing the active links depending on severity of visual deficiency. A deeper shade of blue will be able to accommodate those with low vision and Tritanopia. This also includes the image of text in the logo.

## Detailed Description

Propose adding a color that is greater than 5.6:1 for the active/blue links/text.
